content/en/synthetics/api_tests/http_tests.md

```diff
@@ -21,13 +21,19 @@ further_reading:
 ---
 ## Overview
 
-HTTP tests allow you to **send HTTP requests to your applications' API endpoints** in order to verify that they are responding to requests, as well as that they meet any conditions you define-such as overall response time, status code, and header or body content.
+HTTP tests allow you to send HTTP requests to your applications' API endpoints to verify responses and defined conditions, such as overall response time, status code, and header or body content.
 
-HTTP tests can be run from [**managed**][1] and [**private locations**][2] depending on whether you want to monitor your endpoints from outside or inside your network. HTTP tests can run **on a schedule**, **on-demand**, or directly **within your [CI/CD pipelines][3]**.
+HTTP tests can run from [managed][1] and [private locations][2] from outside or inside your network depending on your preference. HTTP tests can run on a schedule, on-demand, or directly within your [CI/CD pipelines][3].
```

Comment:
I feel like the first sentence makes it less clear that managed locations are outside users' network VS private locations are inside users' network. Can you rework this sentence or put it back to the original one?

Reply:
```suggestion
HTTP tests can run from both [managed][1] and [private locations][2] depending on your preference for running the test from outside or inside your network. HTTP tests can run on a schedule, on-demand, or directly within your [CI/CD pipelines][3].
```